packages/kubernetes-1.22/Cargo.toml

```diff
@@ -14,8 +14,8 @@ path = "pkg.rs"
 package-name = "kubernetes-1.22"
 
 [[package.metadata.build-package.external-files]]
-url = "https://distro.eks.amazonaws.com/kubernetes-1-22/releases/16/artifacts/kubernetes/v1.22.17/kubernetes-src.tar.gz"
-sha512 = "8ff23be77ded11377e0fc0b1ceb6b3bf7ef253e12d59afe9370c81c6cf480cfe0fde0a11a7f23c60f986652c847e3dca06b274ecc01d3f07fbf7330b54927f46"
+url = "https://distro.eks.amazonaws.com/kubernetes-1-22/releases/19/artifacts/kubernetes/v1.22.17/kubernetes-src.tar.gz"
+sha512 = "8a71caf9c490022fe3865bae21b49c56046598004dba524c12cf9ff7954585fb9f97c903539fd491229c0200437462aad46e388c34dd66d22dfa7461a47dfac4"
```

Comment:
Looks like the hash may be wrong?

```
Failed to verify file 'kubernetes-src.tar.gz' with hash '8a71caf9c490022fe3865bae21b49c56046598004dba524c12cf9ff7954585fb9f97c903539fd491229c0200437462aad46e388c34dd66d22dfa7461a47dfac4'
```

Reply:
Hmm, maybe not:

```
$ sha512sum kubernetes-src.tar.gz 
8a71caf9c490022fe3865bae21b49c56046598004dba524c12cf9ff7954585fb9f97c903539fd491229c0200437462aad46e388c34dd66d22dfa7461a47dfac4  kubernetes-src.tar.gz
```

Reply:
Just wondering since the k8s version is still v1.22.17 - what are the actual changes in this version for Bottlerocket?